E508%20Framework.VR/addonsettings.sqf

```diff
@@ -23,81 +23,81 @@ ace_arsenal_EnableRPTLog = false;
-ace_captives_requireSurrender = 1;
-ace_captives_requireSurrenderAi = false;
+force ace_captives_allowHandcuffOwnSide = false;
+force ace_captives_allowSurrender = true;
+force ace_captives_requireSurrender = 1;
+force ace_captives_requireSurrenderAi = false;
 
 // ACE Common
-ace_common_allowFadeMusic = true;
-ace_common_checkPBOsAction = 0;
-ace_common_checkPBOsCheckAll = false;
-ace_common_checkPBOsWhitelist = "[]";
+force ace_common_allowFadeMusic = true;
+force ace_common_checkPBOsAction = 0;
+force ace_common_checkPBOsCheckAll = false;
+force ace_common_checkPBOsWhitelist = "[]";
 ace_common_displayTextColor = [0,0,0,0.1];
 ace_common_displayTextFontColor = [1,1,1,1];
 ace_common_settingFeedbackIcons = 1;
 ace_common_settingProgressBarLocation = 0;
-ace_noradio_enabled = true;
-ace_parachute_hideAltimeter = true;
+force ace_noradio_enabled = true;
+force ace_parachute_hideAltimeter = true;
 
 // ACE Cook off
-ace_cookoff_ammoCookoffDuration = 1;
-ace_cookoff_enable = 2;
-ace_cookoff_enableAmmobox = true;
-ace_cookoff_enableAmmoCookoff = true;
-ace_cookoff_probabilityCoef = 1;
+force ace_cookoff_ammoCookoffDuration = 1;
+force ace_cookoff_enable = 2;
+force ace_cookoff_enableAmmobox = true;
+force ace_cookoff_enableAmmoCookoff = true;
+force ace_cookoff_probabilityCoef = 1;
 
 // ACE Crew Served Weapons
-ace_csw_ammoHandling = 2;
-ace_csw_defaultAssemblyMode = true;
-ace_csw_dragAfterDeploy = true;
-ace_csw_handleExtraMagazines = true;
-ace_csw_progressBarTimeCoefficent = 1;
+force ace_csw_ammoHandling = 2;
+force ace_csw_defaultAssemblyMode = true;
+force ace_csw_dragAfterDeploy = true;
+force ace_csw_handleExtraMagazines = true;
+force ace_csw_progressBarTimeCoefficent = 1;
 
 // ACE Explosives
-ace_explosives_explodeOnDefuse = false;
-ace_explosives_punishNonSpecialists = true;
-ace_explosives_requireSpecialist = true;
+force ace_explosives_explodeOnDefuse = true;
+force ace_explosives_punishNonSpecialists = true;
+force ace_explosives_requireSpecialist = false;
 
 // ACE Fragmentation Simulation
-ace_frag_enabled = true;
-ace_frag_maxTrack = 10;
-ace_frag_maxTrackPerFrame = 10;
-ace_frag_reflectionsEnabled = false;
-ace_frag_spallEnabled = false;
+force ace_frag_enabled = true;
+force ace_frag_maxTrack = 10;
+force ace_frag_maxTrackPerFrame = 10;
+force ace_frag_reflectionsEnabled = false;
+force ace_frag_spallEnabled = false;
 
 // ACE Goggles
-ace_goggles_effects = 2;
-ace_goggles_showClearGlasses = false;
-ace_goggles_showInThirdPerson = false;
+force ace_goggles_effects = 2;
+force ace_goggles_showClearGlasses = false;
+force ace_goggles_showInThirdPerson = false;
 
 // ACE Hearing
-ace_hearing_autoAddEarplugsToUnits = true;
-ace_hearing_disableEarRinging = false;
-ace_hearing_earplugsVolume = 0.5;
-ace_hearing_enableCombatDeafness = true;
-ace_hearing_enabledForZeusUnits = true;
-ace_hearing_unconsciousnessVolume = 0.4;
+force ace_hearing_autoAddEarplugsToUnits = true;
+force ace_hearing_disableEarRinging = true;

```

Comment:
I changed this because I think it's stupid, but idk what people want

Reply:
disagree on this wan - put in earplugs mate; that's what they're there for